msteams-platform/concepts/device-capabilities/device-capabilities-overview.md

```diff
@@ -26,10 +27,15 @@ Use the tools present in [Microsoft Teams JavaScript client SDK](/javascript/api
  
  ## Integrate device capabilities
 
-After getting access to device capabilities,use **Teams media capability APIs** to [integrate the capabilities](mobile-camera-image-permissions.md) with Teams platform to enhance the user experience. These integrated capabilities allow your app to:
+After getting access to device capabilities, use **Teams media capability APIs** to [integrate media capabilities](mobile-camera-image-permissions.md) and use `scanBarCode` API to integrate the [QR or barcode scanner capability](qr-barcode-scanner-capability.md) with Teams platform to enhance the user experience. These integrated capabilities allow your app to:
```

Comment:
Please fix inconsistency and language issues.

Usage of API as bold inconsistently
Remove "use" before scanBarCode API